mlflow/server/js/src/components/ExperimentRunsTableMultiColumnView2.js

```diff
@@ -32,7 +32,7 @@ const EMPTY_CELL_PLACEHOLDER = '-';
 
 export class ExperimentRunsTableMultiColumnView2 extends React.Component {
   static propTypes = {
-    experimentId: PropTypes.number,
+    experimentId: PropTypes.string,
```

Comment:
This is a misc change. Apparently we changed experimentId from an int to a string to be compatible with Azure. This propType was never updated, and was printing a warning.